aws-rds-dbproxyendpoint/aws-rds-dbproxyendpoint.json

```diff
@@ -96,14 +96,22 @@
     "/properties/DBProxyEndpointArn",
     "/properties/Endpoint",
     "/properties/VpcId",
-    "/properties/IsDefault"
+    "/properties/IsDefault",
+    "/properties/TargetRole"
   ],
   "createOnlyProperties": [
     "/properties/DBProxyName",
     "/properties/DBProxyEndpointName",
     "/properties/VpcSubnetIds",
     "/properties/TargetRole"
   ],
+  "tagging": {
+    "taggable": true,
+    "tagOnCreate": true,
+    "tagUpdatable": false,
+    "cloudFormationSystemTags": false,
+    "tagProperty": "/properties/Tags"
+  },
```

Comment:
Why adding this?

Reply:
It was missed in DBProxy but present in DBProxyEndpoint. Adding it here to proxy as well as it supports tagging.

Reply:
We support tag updates with RDS API https://docs.aws.amazon.com/cli/latest/reference/rds/add-tags-to-resource.html. Why ``tagUpdatable`` is false here?